app/views/spree/layouts/admin/_login_nav.html.haml

```diff
@@ -11,4 +11,4 @@
       = link_to t(:logout), spree.logout_path
     %li{"data-hook" => "store-frontend-link"}
       %i.icon-external-link
-      = link_to t(:store), spree.root_path, :target => '_blank'
+      = link_to t("admin.header.store"), spree.root_path, target: "_blank"
```

Comment:
could you check if Rails' [lazy lookup](https://guides.rubyonrails.org/i18n.html#lazy-lookup) would work with `t('.header.store')` here? Using lazy lookup is one of the conventions we have for i18n. See https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-%28i18n%29#development

Reply:
Thanks, @sauloperez. I updated this, and it's now `spree.layouts.admin.header.store` (but referencing through `.header.store`). Right now, I'm questioning whether we should still have `spree` there, but I find it acceptable.